docs/api.rst

```diff
@@ -1,14 +1,14 @@
-Standard API
+Core API
 ============
```

Comment:
❔ Could we adjust the `=` signs under this header so that the two lines have the same length?  It's not required by reStructuredText, but it matches the conventions of the rest of the documentation.